clients/cloud/nodejs/config.js

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const minimist = require('minimist');
+const fs = require('fs');
+const readline = require('readline');
+
+// cli config
+
+const title = "Example Node.js Confluent Cloud client";
+
+const requiredOpts = [
+  ['config', '--config CONFIG', 'The path to your Confluent Cloud configuration file'], 
+  ['topic', '--topic TOPIC', 'The topic name on which to operate']
+];
+
+const requiredConfig = [
+  ['bootstrap.servers', 'bootstrap.servers=<URIs>', 'One or more Kafka bootstrap server addresses. Separate multiple with commas.'],
```

Comment:
nit: the bootstrap servers are not URIs (does not have the scheme: prefix), but simple CSV list of host[:port].

Reply:
Help should probably say "Your Confluent Cloud cluster bootstrap servers..." to be consistent with sasl.username/pass below.

Reply:
Good point - thanks for the correction and suggestion